_blog.yml

```
@@ -3956,3 +3956,13 @@
     - announcement
     - partnerships
     - aws
+- local: langchain_huggingface
```

```suggestion

- local: langchain
```
proposal of better name/URL